### DISCUSSION TITLE
海底捞小程序 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#149] [模板讨论]
海底捞小程序 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by gitcps** [comment]
可以增加tonken模式登录嘛? 并展示币和优惠劵的数据

**Comment by zadishu** [comment]
抓不到这两个值

**Reply by wjf0214** [reply]
小程序退出登录后，开启抓包软件，抓取登录的请求。如果是微信登录，可以看看`login/safeBind`这个请求传递的参数，里面有需要的参数。其他登录方式也是看对应的登录请求就是。
